roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -34,7 +34,7 @@ def run(self, tmp, task_vars):
             }
 
         # attempt to get the docker info hash from the API
-        info = self.execute_module("docker_info", {}, task_vars)
+        info = self.execute_module("docker_info", {}, task_vars=task_vars)
```

Comment:
You're right, although I don't think it breaks anything now, if we get a `tmp` we should pass it on in case anything would use it. Do we need to hold up this PR for that or can it be a separate one?